packages/ui/tests/screenshots/pages/bundles/bundles.base.screen.ts

```diff
@@ -37,6 +39,48 @@ class Bundles extends BasePage {
 
         await this.waitForTableSyncedWidth('.cells-table', {useResizeEvent: true});
     }
+
+    async replaceTabletCells() {
+        await replaceInnerHtml(this.page, {
+            '.cells-table__id-id': '1-XXXX-XXXXX-XXXXXXXXX',
+            '.cells-table__td_col_tablets .cells-table__wrapped': '7',
+            '.cells-table__td_col_memory .cells-table__wrapped': '0 B',
+            '.cells-table__td_col_uncompressed .cells-table__wrapped': '000 KiB',
+            '.cells-table__td_col_compressed .cells-table__wrapped': '000 KiB',
+            '.cells-table__td_col_peeraddress .g-link': 'local:00',
+        });
+        await this.waitForTableSyncedWidth('.data-table', {useResizeEvent: true});
+    }
+
+    async replaceAttributesTime() {
+        await this.page.waitForSelector('pre.unipika span.string');
+        await this.page.evaluate((placeholder) => {
+            const title = document.querySelector('.elements-modal__header span') as HTMLDivElement;
```

Comment:
I think it could be better to do it with generic type, like this: `document.querySelector<HTMLDivElement>`